routers/api/packages/api.go

```diff
@@ -358,6 +358,7 @@ func CommonRoutes() *web.Router {
 					r.Get("/PACKAGES", cran.EnumerateSourcePackages)
```

Comment:
Thank you for submitting this PR. I've tested that I can download from the Archive URL too. Do we need to update the enumeration route to have the archived versions link to the updated URL? Although, if it does, it could be done in another PR after this one is merged.

Reply:
Thanks for the suggestion - I looked at the specs of the official CRAN repository and the /PACKAGES endpoint only supplies information on the newest package versions and not for archived versions. So I don't think we need to update the enumeration route.

Reply:
awesome. thanks for looking into that :)